package.json

```diff
@@ -10,25 +10,25 @@
   },
   "scripts": {
     "analyze": "flow",
-    "clean": "rimraf dist",
+    "build": "bnr build",
```

Comment:
also, glad to have that clean script out. not sure if you've done it already, but having those test configs dynamically built is really hard to debug. I'd prefer duplication for those situations.